src/components/UpdateItem/UpdateItem.tsx

```diff
@@ -144,7 +148,27 @@ export const UpdateItem = ({
             value={formData.itemName}
             onChange={handleChange}
           />
-
+          {categories ? (
```

Comment:
I think this is always gonna be truth, if there are no categories, you will get an empty array [] anyway. You can check if the array is bigger than 0 maybe